pkgs/by-name/hy/hyperhdr/package.nix

```
@@ -38,11 +45,30 @@ stdenv.mkDerivation rec {
     qt6Packages.wrapQtAppsHook
   ];
 
+  patches = [
+    # Allow completly unvendoring hyperhdr
+    # This can be removed on the next hyperhdr release
+    (fetchpatch {
+      name = "USE_SYSTEM_LIBS";
+      url = "https://github.com/awawa-dev/HyperHDR/pull/1161.patch";
+      hash = "sha256-a50RNbbhi40vigGLn1IBy7m5FKET6bbPNhLsoLE+ReI=";
+    })
+  ];
+
+  postPatch = ''
+    substituteInPlace sources/sound-capture/linux/SoundCaptureLinux.cpp \
+      --replace "libasound.so.2" "${alsa-lib}/lib/libasound.so.2"
```

```suggestion
      --replace "libasound.so.2" "${lib.getLib alsa-lib}/lib/libasound.so.2"
```

This will pick a `lib` output, if it exists and makes this more robust against further migrations of the alsa-lib package.